src/components/MessageList/VirtualizedMessageList.tsx

```diff
@@ -0,0 +1,347 @@
+import React, { useCallback, useContext, useMemo, useRef } from 'react';
+import {
+  Components,
+  ScrollSeekConfiguration,
+  ScrollSeekPlaceholderProps,
+  Virtuoso,
+  VirtuosoHandle,
+} from 'react-virtuoso';
+import type { MessageResponse, StreamChat } from 'stream-chat';
+import type { EventComponentProps, TypingIndicatorProps } from 'types';
```

Comment:
we've stopped using the import `from 'types';` and import them from where the type is actually declared now. we're moving towards not having the `index.d.ts`